blockchain/blockchain-wallet-v4-frontend
# feat(FWN-807_809): change cta text

## Description (optional)
### Buy Flow Changes
    Continue → Preview Buy 
    Buy {Asset Name} → Buy Now  
### Sell Flow Changes
    Continue → Preview Sell
    Sell {Asset Name and Asset Amount} → Sell Now 
    
Screenshots
![Screen Shot 2021-11-16 at 10 28 23 AM](https://user-images.githubusercontent.com/39777266/142015093-71048090-5f92-4abd-8e9e-e77ab9f9ef77.png)
![Screen Shot 2021-11-16 at 10 28 35 AM](https://user-images.githubusercontent.com/39777266/142015102-b08b84c9-b372-4dba-9256-18c6c427667a.png)


## Testing Steps (optional)
Login → Buy/Sell Crypto → Select a crypto → make sure title and button have changed
